get_dagmc_model.sh

```diff
@@ -0,0 +1,9 @@
+
+
+# Collimator model
+wget https://anl.box.com/shared/static/isn5mn1y5qf2jjwr4ofnrghf11m6ep3r.h5m -O dagmc.h5m
+mv dagmc.h5m demo_collimator
```

Comment:
```suggestion
mv dagmc.h5m demo_collimator/dagopenmc
```

Reply:
The `dagmc.h5m` should be placed in the `dagopenmc_run`, otherwise, it there will be a file not found error: `ERROR: Geometry DAGMC file 'dagmc.h5m' does not exist!`

Reply:
Nice catch.